controllers/terminal/scripts/Dockerfile-terminal-all

```diff
@@ -0,0 +1,71 @@
+FROM centos:7.6.1810
+# install tty
+RUN yum install -y wget &&  yum install -y bind-utils && \
+    wget https://github.com/tsl0922/ttyd/releases/download/1.4.2/ttyd_linux.x86_64 && \
+    chmod +x ttyd_linux.x86_64 && \
+    mv ttyd_linux.x86_64 /usr/bin/ttyd
+
+# install golang
+RUN cd /root && wget https://dl.google.com/go/go1.12.linux-amd64.tar.gz && \
+	tar zxvf go1.12.linux-amd64.tar.gz && \
+	rm -rf go1.12.linux-amd64.tar.gz && mkdir /root/work
+ENV GOROOT=/root/go GOPATH=/root/work PATH=$PATH:/root/go/bin:/root/work/bin
+
+# ENV APISERVER="https://127.0.0.1:6443"
+# ENV USER_TOKEN="xxx"
+# ENV NAMESPACE="default"
+# ENV USER_NAME="fanux"
+# ENV TERMINAL_ID="UUID"
+
+# install kubectl
+RUN wget https://dl.k8s.io/v1.13.3/kubernetes-client-linux-amd64.tar.gz && \
+    tar zxvf kubernetes-client-linux-amd64.tar.gz && \
+    cp kubernetes/client/bin/kubectl /usr/bin && \
+    rm -rf kubernetes-client-linux-amd64.tar.gz kubernetes
+
+# install zsh
+RUN yum install -y git && \
+	yum -y install zsh &&  sh -c "$(wget https://raw.githubusercontent.com/robbyrussell/oh-my-zsh/master/tools/install.sh -O -)"
+
+# RUN cd /bin && wget https://github.com/neovim/neovim/releases/download/nightly/nvim-linux64.tar.gz && \
+#	tar zxvf nvim-linux64.tar.gz && cp /bin/nvim-linux64/bin/nvim /bin/nvim-linux64/bin/vim && ls /bin/nvim-linux64/bin/vim
+# ENV PATH=$PATH:/bin/nvim-linux64/bin
+
+RUN  yum install -y  ncurses-devel ruby ruby-devel lua lua-devel luajit \
```

Comment:
Do we need ruby/lua/python as default build in software, o just simple base image is enough?

Reply:
@fanux

Reply:
@zzjin User can define its own terminal image, we already provide a simple one.